src/types.ts

```diff
@@ -151,3 +151,8 @@ export interface paletteItemState {
   index: number; // index of the corresponding question in the list of questions
   value: questionState;
 }
+
+export interface UserAPIResponse {
+  _id: string;
+  org_name: string;
```

Comment:
based on my comments in the backend

```suggestion
  name: string;
```